deployment/flatpak/about.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Flatpak
+subsection: flatpak
+section: deployment
+description: >
+	Flatpak (formerly xdg-app) is a software utility for software deployment, package 	management and application virutalization. It provides a sandbox environment in which users 	can run applications in isolation from the rest of the system.
```

Comment:
This is still not fixed. (See https://github.com/developer-portal/content/pull/247#discussion_r163558675)

Reply:
I'm sorry, can you please explain what I'm supposed to fix?
I don't think I'm following. 😅

Reply:
The explanation is linked in the comment.
Please join our #developer-portal irc chanell on [Freenode](https://freenode.net/) so I can explain details there and give you some additional advice.

Reply:
Okay, joining.